server/services/rabbitmq_service.go

```
@@ -0,0 +1,179 @@
+func (r *RabbitMQService) SubscribeWithWorkers(workers int, args SubscribeArgs) {
+	var wg sync.WaitGroup
+
+	log.Printf("[x][%s] Spawning %d workers", args.Queue, workers)
+
+	for i := 1; i <= workers; i++ {
+		wg.Add(1)
+
+		go func() {
+			defer wg.Done()
+
+			r.Subscribe(args)
```

golangci-lint (1.62.2)

Error return value of `r.Subscribe` is not checked

(errcheck)